web/controllers/v1/rule_controller.ex

```diff
@@ -42,6 +42,16 @@ defmodule Cog.V1.RuleController do
     end
   end
 
+  def show(conn, %{"id" => id}) do
+    case Rules.rule(id) do
```

Comment:
This function implementation should be modified to only return a rule if it's currently enabled.